setup.py

```diff
@@ -29,6 +29,9 @@
         'simplejson',
         'six>=1.4.0',
     ],
+    extras_require={
+        'uwsgi_offload': ['uWSGI>=2.0.14.0.post3'],
```

Comment:
Good point. Hmm... the upstream has been pulled in so safe to use 2.0.14 (which is much saner anyways). I'll update.